resources/views/auth/login.blade.php

```diff
@@ -26,16 +27,16 @@
 
         {{-- Email field --}}
         <div class="input-group mb-3">
-            <input type="email" name="email" class="form-control @error('email') is-invalid @enderror"
-                   value="{{ old('email') }}" placeholder="{{ __('adminlte::adminlte.email') }}" autofocus>
+            <input type="{{ $username }}" name="{{ $username }}" class="form-control @error($username) is-invalid @enderror"
+                   value="{{ old($username) }}" placeholder="{{ __('adminlte::adminlte.username') }}" autofocus>
```

Comment:
These changes introduces some bugs, first only a set of possible values are accepted by the input `type` attribute, so it can not be anything, note `$username` variable may hold any string name. Second, the translation entry `__('adminlte::adminlte.username')` does not currently exists.